### PR TITLE
Fix encryption query

### PR DESCRIPTION
**Pull request by jtaleric**
## Type of change

- [ ] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Fixing the encryption query. Tested locally.

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Review by paigerube14 — APPROVED** [review]
/lgtm

File: utils/index.sh [hunk 1]
```diff
@@ -103,7 +103,7 @@ get_encryption_config(){
     # Check the apiserver for the encryption config
     # If encryption was never turned on, you won't find this config on the apiserver
     encrypted=false
-    encryption=$(oc get apiserver -o=json | jq -r '.items[0].spec.encrytion.type')
+    encryption=$(oc get apiserver -o=jsonpath='{.items[0].spec.encryption.type}' )
```
Comment (rsevilla87) [inline utils/index.sh:106]: Maybe you can get the object directly to simplify the expression:

```suggestion
    encryption=$(oc get apiserver cluster -o=jsonpath='{.spec.encryption.type}' )
```

Comment (jtaleric) [reply utils/index.sh:106]: lol let me fire up another clusterbot to check

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**